src/Orleans/Configuration/MessagingConfiguration.cs

```diff
@@ -106,6 +108,11 @@ public interface IMessagingConfiguration
         /// The maximum batch size for automatic batching of messages, when message batching is used.
         /// </summary>
         int MaxMessageBatchingSize { get; set; }
+
+        /// <summary>
+        /// The list of serialization providers
+        /// </summary>
+        HashSet<Type> SerializationProviders { get; }
```

Comment:
`Type` isn't serializable and isn't CoreCLR compatible (https://github.com/dotnet/orleans/pull/973). Replace it with `TypeInfo` instead? Or use type strings in config?